### ISSUE TITLE
Config argument for `team_split_2D` should be const-qualified

### ISSUE THREAD

**Issue by nspark** [gmegan/specification#135]
Current prototype:

```c
int shmem_team_split_2d(shmem_team_t parent_team, int xrange,
    shmem_team_config_t *xaxis_config, long xaxis_mask, shmem_team_t *xaxis_team,
    shmem_team_config_t *yaxis_config, long yaxis_mask, shmem_team_t *yaxis_team);
```

The `xaxis_config` and `yaxis_config` arguments should be `const` qualified.

**Comment by nspark** [comment]
Also, the config arguments are marked as "INOUT" in the argument listing.

**Comment by nspark** [comment]
Closed by #140
